projects/cashmere/src/lib/banner/hc-banner.component.ts

```diff
@@ -11,7 +11,6 @@ import {parseBooleanAttribute} from '../util';
 })
 export class HcBannerComponent {
     private _type: string = 'info';
-    private _fixedTop: boolean = false;
```

Comment:
VS Code indicated that this property wasn't being used anywhere, so I deleted it. See also the `fixed-top` class below.

Reply:
hey @isaaclyman - you're right about this one.  I believe it was added but then we realized it didn't play nice with subnav.  There's an issue logged to make `fixedTop` a directive that can be used across components.  I'll bump that one up on the priority list.  Bump for this component, we're good to remove that orphaned variable

Reply:
This is the issue #654